lexers/internal/api.go

```diff
@@ -104,11 +102,17 @@ func Match(filename string) chroma.Lexer {
 	for _, lexer := range Registry.Lexers {
 		config := lexer.Config()
 		for _, glob := range config.Filenames {
-			if fnmatch.Match(glob, filename, 0) {
+			ok, err := filepath.Match(glob, filename)
+			if err != nil { // nolint
+				panic(err)
```

Comment:
Can you add a test that explicitly calls Match(), to ensure that there are no panics that bubble up?

Reply:
sorry, but I dont really know how to do that.

I tried importing lexers, but I get an import cycle. Then, if I dont import lexers, then I dont get a panic, but I also dont get a lexer back, only nil.

The code builds as is, but I dont know enough about the codebase to make a test I guess. Can you please help out?

Reply:
Aha! Okay no worries, let's merge this and I'll add a test in a followup 🙂